### PR TITLE
Add benchmarks for Grackle + configurable project scalaVersion

### PR DESCRIPTION
**Pull request by kyri-petrou**
Benchmarking results across Caliban 2.3.0, Sangria 4.0.1 and Grackle 0.13.0:

```
[info] Benchmark                             Mode  Cnt      Score      Error  Units
[info] GraphQLBenchmarks.fragmentsCaliban   thrpt    5    127.584 ±    0.576  ops/s
[info] GraphQLBenchmarks.fragmentsGrackle   thrpt    5     68.214 ±    0.915  ops/s
[info] GraphQLBenchmarks.fragmentsSangria   thrpt    5     79.630 ±    1.145  ops/s
[info] GraphQLBenchmarks.introspectCaliban  thrpt    5   4588.751 ±  107.401  ops/s
[info] GraphQLBenchmarks.introspectGrackle  thrpt    5   1814.529 ±    9.122  ops/s
[info] GraphQLBenchmarks.introspectSangria  thrpt    5   1613.585 ±   12.726  ops/s
[info] GraphQLBenchmarks.parserCaliban      thrpt    5  34136.881 ±  675.789  ops/s
[info] GraphQLBenchmarks.parserGrackle      thrpt    5   4484.088 ±   26.300  ops/s
[info] GraphQLBenchmarks.parserSangria      thrpt    5  22734.408 ±  123.280  ops/s
[info] GraphQLBenchmarks.simpleCaliban      thrpt    5  62526.730 ± 1452.906  ops/s
[info] GraphQLBenchmarks.simpleGrackle      thrpt    5  14730.999 ± 1568.441  ops/s
[info] GraphQLBenchmarks.simpleSangria      thrpt    5  16663.193 ±  615.734  ops/s
```

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -36,7 +36,7 @@ val zioPreludeVersion         = "1.0.0-RC19"
 
 inThisBuild(
   List(
-    scalaVersion             := scala212,
+    scalaVersion             := scala212, // Change to control IntelliJ's highlighting
```
Comment (kyri-petrou) [inline build.sbt:39]: When this is changed to `scala213` / `scala3`, intelliJ should pick the right configuration and do highlighting properly

**Comment by ghostdogpr** [comment]
Looks good! I will just give it a try in IntelliJ before merging. Will there be no problem with Metals by the way?

Can you also share the benchmark results in the PR description?

**Comment by kyri-petrou** [comment]
> Will there be no problem with Metals by the way?

Hmmm good point. I don't think so but I'll double check just in case. At least the default SBT configuration should work for Metals just fine

**Comment by ghostdogpr** [comment]
Seems ok in IntelliJ on my side 👍

**Comment by kyri-petrou** [comment]
Still need to check for Metals but I've been busy the past few days and didn't get a chance. Although, I don't use Metals almost ever so I'm not sure whether I'll be able to identify whether something is off or not 🤔

**Comment by ghostdogpr** [comment]
> Still need to check for Metals but I've been busy the past few days and didn't get a chance. Although, I don't use Metals almost ever so I'm not sure whether I'll be able to identify whether something is off or not 🤔

Same here 😄 
@frekw if you're around, I believe you use Metals?

**Comment by frekw** [comment]
I am around now! :)

Yeah I use Metals, let me check!

**Comment by frekw** [comment]
Metals seems to work as usual (which isn't great for Caliban in general – I think it has some problems with the size of the codebase)! 👍

**Comment by kyri-petrou** [comment]
@ghostdogpr was there anything else you wanted to check?

**Review by ghostdogpr — APPROVED** [review]
Good to go for me 👍

**Pull request merged**